cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -11,33 +11,42 @@ var maskedCardNumber;
 const MASKED_CC_PREFIX = '************';
 var oneClickValid = false;
 
-renderCardComponent();
 renderOneClickComponents();
+renderGenericComponent();
+
+function renderGenericComponent(){
+    GetCheckoutPaymentMethods(function(data){
+        var paymentMethodsResponse = JSON.stringify(data.AdyenPaymentMethods);
+
+        var scripts = `
+              <script type="module" src="https://unpkg.com/generic-component@latest/dist/adyen-checkout/adyen-checkout.esm.js"></script>
+              <script nomodule src="https://unpkg.com/generic-component@latest/dist/adyen-checkout/adyen-checkout.js"></script>
+           `;
```

Comment:
this is not something we can launch. Please change this to internal javascript sources.

Reply:
ok agree.

Reply:
Yes, that is correct. Let's change that when merging to develop